src/lrm_scale.rs

```diff
@@ -385,4 +420,34 @@ mod tests {
         assert_eq!(measure.anchor_name, "b");
         assert_eq!(measure.scale_offset, 2.);
     }
+
+    #[test]
+    fn get_measure() {
+        let measure = scale().get_measure(5.).unwrap();
```

Comment:
To stick to previous tests, more readable
`        // a(scale 0)----measure(scale 5)----b(scale 10)----measure(scale 25)
`